client/wp-admin-scripts/print-shipping-label-banner/style.scss

```diff
@@ -1,6 +1,4 @@
 // This is needed so ExternalLinks appear correctly. Should be part of 'wp-components' style but for some reason it's not.
-@import '@wordpress/components/src/visually-hidden/style';
```

Comment:
Removing this like causes problems with the ExternalLink component. They now look like:
![Screenshot from 2020-03-26 15-35-48](https://user-images.githubusercontent.com/1534605/77699169-80263380-6f77-11ea-9096-1ba244480ad1.png)

Reply:
`opens in a new tab` should be hidden. Was there a reason to remove it to fix the issue here?

Reply:
not at all, sorry about that Dave, I removed that line by accident. Thanks for catching this!